src/test/unit_tests/jest_tests/test_nc_account_invalid_mkm_integration.test.js

```diff
@@ -12,7 +12,7 @@ const { TYPES, ACTIONS } = require('../../../manage_nsfs/manage_nsfs_constants')
 const ManageCLIError = require('../../../manage_nsfs/manage_nsfs_cli_errors').ManageCLIError;
 const ManageCLIResponse = require('../../../manage_nsfs/manage_nsfs_cli_responses').ManageCLIResponse;
 
-const tmp_fs_path = path.join(TMP_PATH, 'test_nc_nsfs_account_cli');
+const tmp_fs_path = path.join(TMP_PATH, 'test_nc_account_cli');
```

Comment:
can you change it to `test_nc_invalid_mkm_integration`? it was copied from test_nc_account_cli